erpnext/hr/number_card/employees_left_(last_year)/employees_left_(last_year).json

```diff
@@ -0,0 +1,21 @@
+{
+ "creation": "2020-07-22 11:56:32.947790",
+ "docstatus": 0,
+ "doctype": "Number Card",
+ "document_type": "Employee",
+ "dynamic_filters_json": "[[\"Employee\",\"status\",\"=\",\"frappe.defaults.get_user_default(\\\"Company\\\")\"]]",
```

Comment:
status should be static and the company should be dynamic filter.

Reply:
Done